assets/html/about.html

```diff
@@ -0,0 +1,177 @@
+
+<body>
+  <div class="container">
+    <header>
+      <a href="../../index.html">
+        <img src="../img/logonav.png" id="navlogo" alt="Code for San José logo" />
+      </a>
+      <div id="menu-btn">
+        <i onclick="menu_toggle()" id="fa-btn" class="fas fa-2x fa-bars"></i>
+      </div>
+      <div id="navlinks">
+        <div id="links">
+          <a id="homeLnk" href="../../index.html">Home</a>
+          <a id="volLnk" href="./volunteer.html">Volunteer</a>
+          <a id="abtLnk" href="./about.html">About Us</a>
+          <a id="projLnk" href="./projects.html">Projects</a>
+          <a id="evLnk" href="https://www.meetup.com/Code-for-San-Jose/" target="_blank" rel="noopener noref">Events</a>
+          <a id="donLnk"
+            href="https://secure.codeforamerica.org/page/contribute/donate-to-a-brigade-today?source_codes=Brigade-page&brigade=Code%20for%20San%20Jose"
+            target="_blank" rel="noopener noref">Donate</a>
+          <a id="faqLnk" href="./faq.html">FAQ</a>
+        </div>
+        <div id="sociallinks">
+          <a class="tooltip" href="https://codeforsanjose.slack.com/" target="_blank" rel="noopener noref">
+            <img src="../img/social_logos/Slack_Mark.svg" alt="Slack logo" class="socialnav" />
+            <span class="tooltiptext">Slack</span>
+          </a>
+          <a class="tooltip" href="https://www.meetup.com/Code-for-San-Jose/" target="_blank" rel="noopener noref">
+            <img src="../img/social_logos/meetup-logo-m-swarm.png" alt="Meetup logo" class="socialnav" />
+            <span class="tooltiptext">Meetup</span>
+          </a>
+          <a class="tooltip" href="https://www.facebook.com/codeforsanjose/" target="_blank" rel="noopener noref">
+            <img src="../img/social_logos/f_logo_RGB-Blue_512.png" alt="Facebook logo" class="socialnav" />
+            <span class="tooltiptext">Facebook</span>
+          </a>
+          <a class="tooltip" href="https://twitter.com/codeforsanjose" target="_blank" rel="noopener noref">
+            <img src="../img/social_logos/Twitter_Social_Icon_Circle_Color.svg" alt="Twitter logo" class="socialnav" />
+            <span class="tooltiptext">Twitter</span>
+          </a>
+          <a class="tooltip" href="https://github.com/codeforsanjose/" target="_blank" rel="noopener noref">
+            <img src="../img/social_logos/GitHub-Mark-Light-120px-plus.png" alt="Github logo" class="socialnav" />
+            <span class="tooltiptext">Github</span>
+          </a>
+          <a class="tooltip" href="https://medium.com/code-for-san-jose" target="_blank" rel="noopener noref">
+            <img src="../img/social_logos/Monogram.svg" alt="Medium logo" class="socialnav" />
+            <span class="tooltiptext">Medium</span>
+          </a>
+        </div>
+      </div>
+    </header>
+
+
+    <main>
+      <div class="small-hero" id="about-title">
+        <h1>About Us</h1>
+        <div id="artist-info">
+          <p>Robert Graham</p>
```

Comment:
I see that you wrote this as the title/caption for this image, however, when I view the "About" page, it shows me the placeholder text "Jonny B. Artist @some-social-handle". This actually pops up for all the images associated with small-hero class.

Reply:
For some reason, the top of the image is cut off.

Also, would it be possible to change the description to this?

Plumed Serpent (Quetzalcóatl) 1994, cast stone, 8 x 12 x 12 feet.
Dedicated on November 18, 1996, Plumed Serpent was commissioned by the City of San Jose, California.
Source: https://www.robertgrahamartist.com/plumed-serpent/

Reply:
Changed the description.